cylc/flow/scripts/install.py

```diff
@@ -48,16 +49,34 @@ def main(parser, options, reg):
     for entry_point in pkg_resources.iter_entry_points(
         'cylc.pre_configure'
     ):
-        entry_point.resolve()(Path(flow_file).parent)
+        try:
+            entry_point.resolve()(Path(flow_file).parent)
+        except Exception as exc:
+            # NOTE: except Exception (purposefully vague)
+            # this is to separate plugin from core Cylc errors
+            raise PluginError(
+                'cylc.post_configure',
+                entry_point.name,
+                exc
+            ) from None
 
     for entry_point in pkg_resources.iter_entry_points(
         'cylc.post_install'
     ):
-        entry_point.resolve()(
-            dir_=os.getcwd(),
-            opts=options,
-            dest_root=get_suite_run_dir(suite)
-        )
+        try:
+            entry_point.resolve()(
+                dir_=os.getcwd(),
+                opts=options,
+                dest_root=get_suite_run_dir(suite)
+            )
+        except Exception as exc:
+            # NOTE: except Exception (purposefully vague)
+            # this is to separate plugin from core Cylc errors
+            raise PluginError(
+                'cylc.post_configure',
```

Comment:
Ditto the above, in this case they are slightly different, though I think they may mean the same thing. The plug-ins iterated are `cylc.post_install`, but the error is for `cylc.post_configure`. Just checking if that's intentional, or if it needs updating.